index.bs

```diff
@@ -1109,6 +1109,9 @@ run the following steps:
         then return failure.
     1. [=list/Append=] \``Domain`\`/|domain| ([=encoded=]) to |attributes|.
 1. If |expires| is given, then [=list/append=] \``Expires`\`/|expires| ([=date serialized=]) to |attributes|.
+1. If |path| is not null, then run these steps:
+    1. If |path| does not start with U+0002F (`/`), then return failure.
+    1. If |path| does not end with U+0002F (`/`), append U+0002F (`/`) to |path|.
```

Comment:
style: I've been trying to use "if" paired with "then" explicitly. Add "then" ?

(The spec isn't 100% consistent, though.)